src/molecules/Hero/hero.module.css

```diff
@@ -0,0 +1,16 @@
+.hero {
+  padding: 20px 28px;
```

Comment:
Use custom properties and `calc()` here.

Reply:
Done.